### PR TITLE
add codestyle

### PR DESCRIPTION
**Pull request by hannesa2**
Now, everyone who check this repo out, has the same code style

### CODE REVIEW

**Comment by hannesa2** [comment]
Btw, thank you for the tutorial !

File: .gitignore [hunk 1]
```diff
@@ -14,15 +14,14 @@ classes
...
 
 # gradle
 build
 .gradle
 
 # other
 run
-!LICENSE
-!README.*
-!.gitignore
```
Comment (Cadiboo) [inline .gitignore:27]: Why were these removed?

File: .idea/codeStyles/Project.xml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<component name="ProjectCodeStyleConfiguration">
+  <code_scheme name="Project" version="173">
+    <option name="RIGHT_MARGIN" value="150" />
+  </code_scheme>
+</component>
```
Comment (Cadiboo) [inline .idea/codeStyles/Project.xml:5]: This simply adds a default code style with nothing in it, effectively basing it off whatever the user already had/IntelliJ's default

**Comment by Cadiboo** [comment]
Thanks for the PR, but as some people might not use IntelliJ + the code style you've added is pretty much empty, I'm not going to merge this.

Also, in my view I shouldn't be enforcing my code style on other people in their own projects, and this example mod is meant to be used as a reference of good code, not as "my project"

File: .gitignore (hunk 1 above)
Comment (hannesa2) [reply .gitignore:27]: It's simply noise. They are added by default.
And make an exception of ignore = add it = default

File: .idea/codeStyles/Project.xml (hunk 2 above)
Comment (hannesa2) [reply .idea/codeStyles/Project.xml:5]: With this, it uses the checked-in "Project", otherwise there is noting applied.

File: .gitignore (hunk 1 above)
Comment (Cadiboo) [reply .gitignore:27]: Do you mean that by default `.gitignore` includes these files and these lines aren't necessary to have them be tracked by git? Or do you mean that you think they shouldn't be included in peoples repos?

File: .idea/codeStyles/Project.xml (hunk 2 above)
Comment (Cadiboo) [reply .idea/codeStyles/Project.xml:5]: > With this, it uses the checked-in "Project", otherwise there is noting applied.

Can you explain this more please?

File: .gitignore (hunk 1 above)
Comment (hannesa2) [reply .gitignore:27]: "_these lines aren't necessary to have them be tracked by git_"

Comment (Cadiboo) [reply .gitignore:27]: Thanks, I’ll check that and remove them if they aren’t needed, I seem to remember that they were necessary
